packages/aws-cdk/lib/settings.ts

```diff
@@ -47,6 +47,22 @@ export type Arguments = {
   readonly [name: string]: unknown;
 };
 
+export interface ConfigurationProps {
```

Comment:
Are these changes related to the SSM/v5 changes at all? They seem unrelated.

Reply:
They are the result of an other issue that I found while working on this. Separate PR here: https://github.com/aws/aws-cdk/pull/12579

These changes on this branch will vanish if that other PR is merged.